event-stream-service/charts/event-stream-service/values-dev.yaml

```
@@ -0,0 +1,14 @@
+nats:
+  config:
+    cluster:
+      replicas: 3
+    websocket:
+      enabled: true
+      port: 8888
+  configMap:
+    merge:
+      data:
+        params.conf: |
+          max_payload: 1MB
```

The `reloader` container doesn't have resources defined, so it's using the default: 50m->250m CPU, 256Mi->1Gi memory. I tried tuning it down in dev and it seems to function fine with: 

```suggestion
          max_payload: 1MB
  reloader:
    merge:
      resources:
        limits:
          cpu: 50m
          memory: 64Mi
        requests:
          cpu: 10m
          memory: 32Mi

```

Editing the configmap the logs show:
```
2024/11/21 22:37:34 Starting NATS Server Reloader v0.16.0
2024/11/21 22:37:34 Watching file: /etc/nats-config/nats.conf
2024/11/21 22:37:34 Live, ready to kick pid 7 on config changes (files=1)
2024/11/21 22:39:21 Starting ticker to re-add all tracked files.
2024/11/21 22:39:21 Changed config; file="/etc/nats-config/nats.conf" existing=true total-files=1
2024/11/21 22:39:21 Updated files: [/etc/nats-config/nats.conf]
2024/11/21 22:39:21 Sending pid 7 'hangup' signal to reload changes from: [/etc/nats-config/nats.conf]
2024/11/21 22:39:22 Tracking files [/etc/nats-config/nats.conf]
2024/11/21 22:39:22 All monitored files detected.
```